### ISSUE TITLE
add a activeContainerNode

### ISSUE THREAD

**Issue by btopro** [LRNWebComponents/hax-body#98]
This will help with getting the grid plate separated from the activenode because there would be (effectively) an active grid plate and then an active element. The active element could be any valid element on the page but the active grid plate / activeContainerNode would be in charge of providing a place visually for the context pallet to reside as well as moving the entire plate. This also would set things up better for #74 and wouldn't be as aggressive a change as previously thought.

Effectively we'd just have to position whenever that node changes and when active changes / is hidden we'd still remove the menus

**Comment by btopro** [comment]
```
activeParentNode

activeParentNode gets us the high level element placement, important for many reasons

activeParentNode would loop through at the time we are finidng the highest and
see if anything matches below that. The last valid match that IS NOT at the highest level, we keep as a activeNode.

activeParentNode is what the grid plate placement is based on as well as where to add the contextual menu

activeNode is what is still used to supply the settings

Basically just abstracts the node to target for placing the items from the node to target for actually working against.
```

**Comment by btopro** [comment]
whew, that's a huge step forward
